packages/web/app/tailwind.config.cjs

```diff
@@ -70,10 +71,29 @@ module.exports = {
         700: '#008c66',
         800: '#007052',
         900: '#005b43',
+        // workaround for ESLint not respecting Tailwind settings paths in CI
```

Comment:
I'm confused why ESLint doesn't find the proper Tailwind config on CI, and this seems like something that will solve itself in time: I assume that eventually the app and the website will use the same color palette.

Reply:
cc @dimaMachina to help here